Library/Homebrew/bottle_version.rb

```diff
@@ -63,3 +65,25 @@ def self._parse spec
     super
   end
 end
+
+class BottlePkgVersion < PkgVersion
```

Comment:
BottleVersion only parses the formula version but not revision part. So a new class to parse revision as well.